akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -129,6 +129,7 @@ object Member {
    * INTERNAL API
    * Orders the members by their address except that members with status
    * Joining, Exiting and Down are ordered last (in that order).
+   * FIXME include shutting down
    */
   private[cluster] val leaderStatusOrdering: Ordering[Member] = Ordering.fromLessThan[Member] { (a, b) =>
```

Comment:
this wasn't changed, but that seems correct to me. We don't want to change leader because of PreparingForShutdown and ReadyForShutdown. They are equal to Up in this regard.